package/network/config/wifi-scripts/files/lib/netifd/hostapd.sh

```diff
@@ -1409,7 +1409,10 @@ wpa_supplicant_add_network() {
 			case "$eap_type" in
 				tls)
 					json_get_vars client_cert priv_key priv_key_pwd
-					append network_data "client_cert=\"$client_cert\"" "$N$T"
+     					# When PKCS#12/PFX file (.p12/.pfx) is used, client_cert should be commented out
+     					if [[ -n "$priv_key" && "$priv_key" != *".p12"* && "$priv_key" != *".pfx"* ]]; then
```

Comment:
`[[` and `]]` is bash syntax. Don't use it. Globs within `[` and `]` also no go. Needs a `case...esac`.
Suggest
```sh
if [ -n "$priv_key" ]; then
	case "$priv_key" in
	 *".p12")
		;;
	 *".pfx")
		;;
	 *)
	 append ...
	esac
fi
```